examples/rpc-db/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "rpc-db"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+eyre = "0.6.8"
+futures.workspace = true
+jsonrpsee.workspace = true
+reth-beacon-consensus.workspace = true
+reth-blockchain-tree.workspace = true
+reth-db.workspace = true
+reth-network-api.workspace = true
+reth-primitives.workspace = true
+reth-provider.workspace = true
+reth-revm.workspace = true
+reth-rpc = { version = "0.1.0-alpha.8", path = "../../crates/rpc/rpc" }
+reth-rpc-builder.workspace = true
+reth-tasks.workspace = true
+reth-transaction-pool.workspace = true
```

Comment:
I don't think we need all of this most of this is reexported from reth itself

https://github.com/paradigmxyz/reth/blob/685148117aab83d4b38f2e05adb8c6c96943f7db/examples/additional-rpc-namespace-in-cli/Cargo.toml#L8-L14

Reply:
Had to export a bit of stuff, but should be good now!